jarbas/public_admin/tests/test_public_admin_site.py

```diff
@@ -5,15 +5,15 @@
 from django.contrib.contenttypes.models import ContentType
 from django.test import TestCase
 
-from jarbas.dashboard.sites import DashboardSite, DummyUser, dashboard
+from jarbas.public_admin.sites import PublicAdminSite, DummyUser, public_admin
```

Comment:
🎉  shouldn't this import line be `from jarbas.public_admin.sites import DummyUser, PublicAdminSite, public_admin`?

Reply:
Fixed in 94d8f94